### PR TITLE
Acrolinx 4/13

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/csharp/how-to/concatenate-multiple-strings.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ In some expressions, it's easier to concatenate strings using string interpolati
 
 Other method to concatenate strings is <xref:System.String.Format%2A?displayProperty=nameWithType>. This method works well when you are building a string from a small number of component strings.
 
-In other cases you may be combining strings in a loop, where you don't know how many source strings you are combining, and the actual number of source strings may be quite large. The <xref:System.Text.StringBuilder> class was designed for these scenarios. The following code uses the <xref:System.Text.StringBuilder.Append%2A> method of the <xref:System.Text.StringBuilder> class to concatenate strings.  
+In other cases, you may be combining strings in a loop where you don't know how many source strings you are combining, and the actual number of source strings may be large. The <xref:System.Text.StringBuilder> class was designed for these scenarios. The following code uses the <xref:System.Text.StringBuilder.Append%2A> method of the <xref:System.Text.StringBuilder> class to concatenate strings.
```
Comment (tdykstra) [inline docs/csharp/how-to/concatenate-multiple-strings.md:34]:
```suggestion
In other cases, you may be combining strings in a loop where you don't know how many source strings you're combining, and the actual number of source strings may be large. The <xref:System.Text.StringBuilder> class was designed for these scenarios. The following code uses the <xref:System.Text.StringBuilder.Append%2A> method of the <xref:System.Text.StringBuilder> class to concatenate strings.  
```

File: docs/machine-learning/tutorials/sentiment-analysis-model-builder.md [hunk 2]
```diff
@@ -99,9 +99,9 @@ The time required for the model to train is proportionate to the amount of data.
 
 ## Evaluate the model
 
-The result of the training step will be one model which had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the **Best Model** entry along with metrics in **Best Model Accuracy**. Additionally, a summary table containing top five models and their metrics.
+The result of the training step will be one model that had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the **Best Model** entry along with metrics in **Best Model Accuracy**. Additionally, a summary table containing top five models and their metrics.
```
Comment (tdykstra) [inline docs/machine-learning/tutorials/sentiment-analysis-model-builder.md:102]:
```suggestion
The result of the training step will be one model that has the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the **Best Model** entry along with metrics in **Best Model Accuracy**. Additionally, a summary table containing top five models and their metrics.
```

**Review by natke — APPROVED** [review]
The model builder changes look good to me!

**Review by tdykstra — APPROVED** [review]
LGTM

**Comment by gewarren** [comment]
Thanks for the quick reviews!

**Pull request merged**